.github/workflows/ci.yml

```diff
@@ -171,11 +102,21 @@ jobs:
     steps:
       - name: "Checking out repository"
         uses: actions/checkout@e2f20e631ae6d7dd3b768f56a5d2af784dd54791 # v2.5.0
+      # Restore caches unconditionally. Cache misses are okay.
```

Comment:
I mean isn't the definition of it being a cache that cache misses are ok? :-P I just read up on the cache action and the names of these fields aren't super intuitive to me. Maybe it's worth a comment to note that it will first try to grab the cache for this exact commit falling back to the latest for any commit?